### ISSUE TITLE
Chrome reports "Your connection has been interrupted" constantly

### ISSUE THREAD

**Issue by djmarcin** [tailscale/tailscale#3686]
### What is the issue?

For about the last week, multiple users on our network have been receiving the error message on chrome:

```
Your connection was interrupted
A network change was detected.
ERR_NETWORK_CHANGED
```

This only seems to happen when Tailscale is running. I have only observed the issue on macOS but do not know that it is strictly limited to macOS.

### OS

macOS

### OS version

macOS Big Sur 11.6

### Tailscale version

1.18.2

### Bug report

BUG-17f9d210538ef6d2db9fbc248d2c5d9c3a49b6a73b867fa0cf353a30b96dce2d-20220109192612Z-4b49040fd77fc49a

**Comment by bradfitz** [comment]
Chrome detects network changes and aggressively reconnects whenever it sees any network change.

From a quick glance, your network's 100.73.102.206 Ubuntu Focal node on AWS is restarting a bunch and looks unhappy somehow.

**Comment by bradfitz** [comment]
Yeah, that 100.74.102.206 seems to be in a fight with another machine using the same key...

```
2022-01-08 17:38:30.354543 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514" to "discokey:431dfc7f4f6956e9119115d76071817f94582853a58c67c16b246dc97668dc65"
2022-01-08 17:38:49.291079 -0800 PST: magicsock: disco: node [7IvJQ] changed from discokey discokey:8841be96697d5ca95d27ef046a5ba5771fe79bafc2d4dc0d56cc264586618d51 to discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514
2022-01-08 17:38:49.291929 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:8841be96697d5ca95d27ef046a5ba5771fe79bafc2d4dc0d56cc264586618d51" to "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514"
2022-01-08 17:39:04.841978 -0800 PST: magicsock: disco: node [7IvJQ] changed from discokey discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514 to discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41
2022-01-08 17:39:04.842709 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514" to "discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41"
2022-01-08 17:39:33.347911 -0800 PST: magicsock: disco: node [7IvJQ] changed from discokey discokey:8841be96697d5ca95d27ef046a5ba5771fe79bafc2d4dc0d56cc264586618d51 to discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514
2022-01-08 17:39:33.348647 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:8841be96697d5ca95d27ef046a5ba5771fe79bafc2d4dc0d56cc264586618d51" to "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514"
2022-01-08 17:39:48.383763 -0800 PST: magicsock: disco: node [7IvJQ] changed from discokey discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514 to discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41
2022-01-08 17:39:48.384314 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514" to "discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41"
2022-01-08 17:40:32.96174 -0800 PST: magicsock: disco: node [7IvJQ] changed from discokey discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41 to discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514
2022-01-08 17:40:32.96246 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41" to "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514"
2022-01-08 17:40:49.546215 -0800 PST: magicsock: disco: node [7IvJQ] changed from discokey discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514 to discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41
2022-01-08 17:40:49.546956 -0800 PST: wgengine: Reconfig: [7IvJQ] changed from "discokey:be228d14cbd31d4a191af48ad12c9671db4fe4afc425bb9349e688188b397514" to "discokey:0c11163dcfec2b529f6eb081233333a175762501a883eaab3eec8cf3a0d8cc41"
```

I suspect somebody cloned a VM and there are two `tailscaled` processes running trying to be the same wireguard key.

**Comment by djmarcin** [comment]
Yes, that definitely happened. What is the process to fix that, --force-reauth on one of them?

**Comment by bradfitz** [comment]
There's other state they're sharing besides that, so I'd do:

```
# systemctl stop tailscaled
# rm -rf /var/lib/tailscale
# systemctl start tailscaled
```

And then `tailscale up`.

We're working on detecting this automatically and doing the right thing in many cases.

**Comment by bradfitz** [comment]
Filed #3706 for tracking making the web admin console show when this is happening.

**Comment by DentonGentry** [comment]
I think this one is resolved and followup work can be handled in #3706.
